libmscore/select.h

```diff
@@ -130,6 +130,9 @@ class Selection {
       Segment* _startSegment;
       Segment* _endSegment;         // next segment after selection
 
+      int _plannedTick1 = -1; // will be actually selected on updateSelectedElements() call
```

Comment:
`_plannedTick`? Absolutely not clear what this variable does, unfortunately. I suppose it somehow defines begin/end range ticks to be selected after some operation, doesn't it? But better name or explaining comment will help to get the meaning better.